hw/ip/spi_device/data/spi_device.hjson

```diff
@@ -43,6 +43,28 @@
     { name: "tx_underflow"
       desc: "TX Async FIFO underflow"
     }
+    { name: "cmdfifo_not_empty"
```

Comment:
@eunchan do you think there's a need to prefix or explain elsewhere in the doc which mode can generate which interrupt?

Reply:
I think it is clearer than current one. Let me address the new intr name in a new PR.